tests/flow/test_hnsw_parallel.py

```diff
@@ -228,7 +237,6 @@ def test_parallel_with_range():
     radius = 3.0
     n_threads = min(os.cpu_count(), 8)
     PADDING_LABEL = -1  # used for padding empty labels entries in a single query results
-    expected_parallel_rate = 0.9  # we expect that at least 90% of the insert/search time will be executed in parallel
```

Comment:
canbe removed also from `test_parallel_insert` `test_parallel_search` and `test_parallel_insert_multi `